packages/expo-av/src/AV.types.ts

```diff
@@ -245,6 +249,11 @@ export type AVPlaybackStatusToSet = {
    * > Note that this only affect the audio of this `playbackObject` and do NOT affect the system volume.
    */
   isMuted?: boolean;
+  /**
+   * The current audio panning value of the audio for this media.
+   * > Note that this only affect the audio of this `playbackObject` and do NOT affect the system volume.
+   */
+  audioPan?: number;
```

Comment:
For user-settable properties, you can mark them as platform-specific 😉 
```suggestion
  /**
   * The current audio panning value of the audio for this media.
   * > Note that this only affect the audio of this `playbackObject` and do NOT affect the system volume.
   * @platform android
   */
  audioPan?: number;
```